rllib/utils/metrics/stats/utils.py

```diff
@@ -0,0 +1,14 @@
+from ray.rllib.utils.framework import try_import_torch, try_import_tf
+from ray.util.annotations import DeveloperAPI
+
+torch, _ = try_import_torch()
+_, tf, _ = try_import_tf()
+
+
+@DeveloperAPI
+def single_value_to_cpu(value):
+    if torch and isinstance(value, torch.Tensor):
```

Comment:
Where is this intended to be called? If we are running Torch DDP on GPUs any call to `.cpu()` synchronizes on GPU could interfere with CUDA synchronizations during updates.

Reply:
This is intended to be called every time we add a value to the metrics logger.

@kamil-kaczmarek and me talked about this last Friday and decided to try always moving values to CPU and re-introduce other GPU tensor handling only if there is a regression.

Are you foreseeing that this will definitely happen?